change/@fluentui-react-charting-fdcd995e-d8b9-4952-a4a2-2674d8757d49.json

```diff
@@ -0,0 +1,7 @@
+{
```

Comment:
## [🕵 fluentuiv8 Open the Visual Regressions report to inspect the affected screenshots]( https://staticpagerelease.z16.web.core.windows.net/azurestaticapps/vr-approval/index.html?organization=uifabric&project=fabricpublic&prId=29370&commitId=443c197ea4887f170b933409f6e47eb13919325a&env=prod&clientType=FLUENTUI)


  <details><summary><b>MediaTile</b> 1 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|MediaTile.Media tile with nameplate hidden until hover.hover.chromium.png|175| Changed|

</details>


  <details><summary><b>react-charting-TreeChart</b> 6 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|react-charting-TreeChart.Compact - Dark Mode.default.chromium.png|115228| Changed|
|react-charting-TreeChart.Basic.default.chromium.png|91059| Changed|
|react-charting-TreeChart.Basic - RTL.default.chromium.png|90986| Changed|
|react-charting-TreeChart.Basic - Dark Mode.default.chromium.png|109565| Changed|
|react-charting-TreeChart.Compact - RTL.default.chromium.png|91288| Changed|
|react-charting-TreeChart.Compact.default.chromium.png|91385| Changed|

</details>




<div id="vrtCommentfluent-ui_VRT_Pipeline_v8"/>